optimizationProblem.m

```diff
@@ -15,23 +15,29 @@
     
     properties (Access = public)
         targetTensor = 1/3*eye(3); %Isotropic encoding tensor
-        N = 50;
+        N = 77;
         initialGuess = 'random';
         useMaxNorm = true;
         gMax = 80;
         sMax = 100;
-        durationFirstPartRequested = 25;
-        durationSecondPartRequested = 25;
-        durationZeroGradientRequested = 0;
-        eta = 1;
+        durationFirstPartRequested = 28;
+        durationSecondPartRequested = 22;
+        durationZeroGradientRequested = 8;
+        eta = .7;
         enforceSymmetry = false;
         redoIfFailed = true;
         name = 'NOW';
+        x0 = [];
+        doMaxwellComp = 1;
```

Comment:
Maybe there should be a way to toggle this off, although it could be on by default.